app/assets/stylesheets/_spacing-units.scss

```diff
@@ -0,0 +1,9 @@
+$su1: 4px;
```

Comment:
I feel like this is the opportunity to start using em/rem?

Reply:
Similarly as in colors: these are not imported anywhere, so they are not ready to use.  Definitely agree we could use rems but not sure if for spacing units (spacing units are meant to be used for paddings, margins etc.. and `rem`s should be probably used with typography system. Although, we could use them here as well, why not :)

The reason why I'm not basing it on rems RIGHT NOW is because `rem` value is based on font-size in `body` - and that one is currently not the one we'd like to be. And fixing that will affect looots of stuff and I just don't wanna do it right now :D